vscode-extension/src/prompts/workItemInitPrompt.template.md

```diff
@@ -0,0 +1,172 @@
+# PAW Work Item Initialization
+
+You are tasked with creating a complete PAW (Phased Agent Workflow) work item directory structure.
+
+## Parameters Provided
+
+- **Target Branch**: {{TARGET_BRANCH}}
+- **Issue URL**: {{ISSUE_URL}}
+- **Workspace Path**: {{WORKSPACE_PATH}}
+
+{{CUSTOM_INSTRUCTIONS}}
+
+## Your Tasks
+
+### 1. Generate Feature Slug
+
+Normalize the target branch name into a valid feature slug following PAW rules:
+
+**Normalization Steps:**
+1. Remove standard prefixes like "feature/", "bugfix/", or "hotfix/" to extract the descriptive portion
+2. Convert the remaining text to lowercase
+3. Replace spaces and special characters with hyphens
+4. Remove any characters that are not lowercase letters, numbers, or hyphens
+5. Collapse consecutive hyphens into a single hyphen
+6. Trim leading and trailing hyphens
+7. Enforce a maximum length of 100 characters
+
+**Validation Requirements:**
+- Length must be between 1 and 100 characters
+- Allowed characters: lowercase letters (a-z), numbers (0-9), hyphens (-) only
+- No leading or trailing hyphens
+- No consecutive hyphens
+- Disallow reserved names: ".", "..", "node_modules", ".git", ".paw"
+
+**Uniqueness Check:**
+- Verify that `.paw/work/<slug>/` does not already exist
+- If a conflict is detected, append -2, -3, etc. until a unique slug is found
+
+**Normalization Examples:**
+| Input | Normalized Slug |
+|-------|-----------------|
+| "User Authentication System" | user-authentication-system |
+| "API Refactor v2" | api-refactor-v2 |
+| "Fix: Rate Limit Bug" | fix-rate-limit-bug |
+| "my_FEATURE--test" | my-feature-test |
+
+### 2. Generate Work Title
+
+{{WORK_TITLE_STRATEGY}}
+
+**Branch-Based Generation{{WORK_TITLE_FALLBACK_INDICATOR}}:**
+- Remove prefixes such as feature/, bugfix/, or hotfix/
+- Split the remaining name on hyphens, underscores, and slashes
+- Capitalize the first letter of each word
+- Join words with spaces and keep the title concise (ideally 2-4 words)
+- Example: feature/user-auth-system → User Auth System
+
+### 3. Create Directory Structure
+
+Create the following structure within the workspace:
+
+```
+.paw/work/<feature-slug>/
+├── WorkflowContext.md
+└── prompts/
+    ├── 01A-spec.prompt.md
```

Comment:
These prompt files are created by a tool call. Do not place them here, as the agent doesn't need to know about it - it just needs to create the empty prompts directory.